### ISSUE TITLE
Show all relevant contact info fields on /myesp/profile

### ISSUE THREAD

**Issue by willgearty** [learning-unlimited/ESP-Website#2748]
Right now, `/myesp/profile` picks a single user type (currently Teacher -> Student ->Guardian -> Educator -> Volunteer) to use for populating the profile form, however, many users have multiple user types. For example, a former student may now be a teacher, or a guardian may also be an educator, etc (and, of course, admins who are often many user types for testing purposes). Within the program-specific profile editor, it's probably fine to focus on the user type related to the registration type (student or teacher), but for `/myesp/profile` we should really be showing all of the relevant contact info fields. Otherwise, as soon as a user has multiple user types they will never be able to edit the info for the fields for the older/lower priority user types.

Unfortunately, I don't think this will be a super easy fix. We'll need to switch from the user type being a single string to being a list/set which we iterate over. Also, some of the fields in the different contact infos have the same names (for example, student graduation year and teacher graduation year), so we'll need to come up with a way to prevent those from conflicting when the form is submitted.

**Comment by DarshanCode2005** [comment]
Hi Dr. @willgearty, I noticed this issue is still open but labeled as “Idea for Discussion.” I’d be interested in working on it if it’s still relevant for Stable Release 17. Before starting, do you have a preferred approach in mind for handling multiple user types (e.g., moving from a single user_type string to a list/set)? I’m also happy to draft a short proposal outlining how we might handle field conflicts before implementing anything. Let me know, if you are willing to pass on the assignment to me.

**Comment by willgearty** [comment]
@DarshanCode2005 Thanks for your interest in contributing to our project! I've assigned you to this issue. Please make sure to check out our [instructions for getting started](https://github.com/learning-unlimited/ESP-Website/wiki#getting-started) and follow our [contribution rules](https://github.com/learning-unlimited/ESP-Website/wiki#rules). Let me know if you have any questions.
